### ISSUE TITLE
Remove keyboard traps from non-modal views

### ISSUE THREAD

**Issue by quincylvania** [openstreetmap/iD#7770]
A keyboard trap occurs when a user cannot navigate away from a UI element via the keyboard alone. This is an [accessibility problem](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-trapping.html).

Modal views can restrict the keyboard focus, since their purpose is to temporarily limit the scope of the UI. But the modal must still be escapable/resolvable using the keyboard alone.

Re: #7713
